Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-Failed SQL Logons.yaml

```diff
@@ -1,6 +1,10 @@
 id: d98256d5-0c9a-4ffc-8618-66a3404412f8
 name: Failed Logon Attempts on SQL Server
 description: |
+    This query detects failed logons on SQL Server using the SQLEvent KQL Parser function. 
```

Comment:
This is enough "This query detects failed logons on SQL Server using the SQLEvent KQL Parser function."